### ISSUE TITLE
New update broke Hyprland

### ISSUE THREAD

**Issue by JaKooLit** [hyprwm/Hyprland#2914]
### Hyprland Version

hyprland-nvidia-git and hyprland-git v0.28.0-13-gcebab759

### Bug or Regression?

Bug

### Description

After update, cannot login. Just a brief white screen, and immediately back to sddm. Logged in through tty and attempted to launch hyprland by typing Hyprland, same issue

This is same issue with both hyprland-nvidia-git and hyprland-git AUR packages.

I have installed hyprland from extra repo just to log in back
[hyprlandCrashReport949.txt](https://github.com/hyprwm/Hyprland/files/12290366/hyprlandCrashReport949.txt)

See attached last hyprland crash report

### How to reproduce

install hyprland-git or hyprland-nvidia-git AUR packages?

### Crash reports, logs, images, videos

[hyprlandCrashReport949.txt](https://github.com/hyprwm/Hyprland/files/12290372/hyprlandCrashReport949.txt)

**Comment by VincentBerthier** [comment]
Apparently there’s a PR that solves the issue: #2908

**Comment by JaKooLit** [comment]
hmmmm.. but this happened after PR was merged. Latest merged was 16 hrs ago and git packages from aur has and upstream to this repo

I'll try later on building from source if same issue

**Comment by maaarghk** [comment]
No, #2905 was merged and introduced the issue. #2908 is not merged and will fix the issue.

**Comment by NekoCWD** [comment]
Now all works fine

**Comment by JaKooLit** [comment]
> Now all works fine

yup just updated... its all worked fine now.

Im closing this as completed

Thanks to all devs especially to lord Vaxry
